pythainlp/tokenize/thaisumcut.py

```diff
@@ -0,0 +1,343 @@
+# -*- coding: utf-8 -*-
+"""
+The implementation of sentence segmentator from Nakhun Chumpolsathien, 2020
+original code from: https://github.com/nakhunchumpolsathien/ThaiSum
+
+Cite:
+
+@mastersthesis{chumpolsathien_2020, 
+    title={Using Knowledge Distillation from Keyword Extraction to Improve the Informativeness of Neural Cross-lingual Summarization},
+    author={Chumpolsathien, Nakhun}, 
+    year={2020}, 
+    school={Beijing Institute of Technology}
+"""
+
+import re
+import pandas as pd
+import pythainlp
```

Comment:
Can you use replace `import pythainlp` with `from pythainlp.tokenize import word_tokenize`?